indico/htdocs/sass/partials/_forms.scss

```diff
@@ -205,6 +205,7 @@ $i-form-button-line-height: 1.8;
     textarea {
         @extend %form-field-input;
         height: auto;
+        width: 100%;
```

Comment:
Make sure this doesn't break the layout anywhere - it seems to affect almost every textarea in indico.